### ISSUE TITLE
Missing info

### ISSUE THREAD

**Issue by Killbox67** [MeSoftHorny/WTRTI#42]
Currently there is no displayed information in the Mig 29 SMT model in regards to fuel consumption and the fuel load duration
![imagem_2023-10-09_010659391](https://github.com/MeSoftHorny/WTRTI/assets/147353338/9912bcf9-b7fa-4ee6-b5b7-b69d00e50682)

**Comment by MeSoftHorny** [comment]
Yes, it's a known issue, unfortunately there's nothing I can do about it. 
The "fuel" data is missing in localhost:8111, so only Gajin's can fix it.
